Formula/libglib.rb

```diff
@@ -0,0 +1,142 @@
+class Libglib < Formula
+  # Sync everything from Formula/glib.rb
+  def self.libglib?
+    name.end_with?("Libglib")
+  end
+
+  include Language::Python::Shebang unless libglib?
+
+  desc "Core application library for C"
+  homepage "https://developer.gnome.org/glib/"
+  url "https://download.gnome.org/sources/glib/2.72/glib-2.72.1.tar.xz"
+  sha256 "c07e57147b254cef92ce80a0378dc0c02a4358e7de4702e9f403069781095fe2"
+  license "LGPL-2.1-or-later"
+
+  if libglib?
+    livecheck do
+      formula "glib"
+    end
+  end
+
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => :build
+
+  if libglib?
+    depends_on "python@3.9" => :build
+    depends_on "gettext"
+    depends_on "libffi"
+    depends_on "pcre"
+  else
+    depends_on "libglib"
+    depends_on "python@3.9"
+  end
+
+  on_linux do
+    depends_on "util-linux"
+  end
+
+  if libglib?
+    link_overwrite "lib"
```

Comment:
Not sure how to use `link_overwrite` with directory or glob pattern.

Should we add `link_overwrite` for each individual file here?

https://github.com/Homebrew/homebrew-core/runs/6538034303?check_suite_focus=true#step:11:1835